airflow/operators/python.py

```diff
@@ -145,6 +145,69 @@ def execute_callable(self):
         return self.python_callable(*self.op_args, **self.op_kwargs)
 
 
+class PythonFunctionalOperator(PythonOperator):
```

Comment:
[discussion] There is probably not too much value for inheriting from PythonOperator
( there is not too much common logic) and .execute is completely overridden,
at the same time,  it defines all "required" features for running python function in Airflow. 
it's also going to be easier to unwrap PythonOperator into @task, and that's mean that we will have priority for supporting all kwargs of PythonOperator

Reply:
I agree with @evgenyshulman , inheritance in case of operators can lead to strange problems (i.e. overwritten template files that break underlying functionality - a bug from yesterday) and is rather discouraged (except "Base" operators) .

Reply:
Switched to use BaseOperator as suggested.